.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:8.15.1
```

Comment:
We should use node lts

Reply:
As per our talk, we will use this image as it doesn't break `npm install` after restoring cache